platforms/gke/base/core/deploy.sh

```diff
@@ -41,48 +45,14 @@ if [ ${STATE_MIGRATED} == "false" ]; then
     rm -rf state
 fi
 
-cd ${ACP_PLATFORM_CORE_DIR}/networking &&
-    echo "Current directory: $(pwd)" &&
-    terraform init &&
-    terraform plan -input=false -out=tfplan &&
-    terraform apply -input=false tfplan || exit 1
-rm tfplan
-
-cd ${ACP_PLATFORM_CORE_DIR}/container_cluster &&
-    echo "Current directory: $(pwd)" &&
-    terraform init &&
-    terraform plan -input=false -out=tfplan &&
-    terraform apply -input=false tfplan || exit 1
-rm tfplan
-
-cd ${ACP_PLATFORM_CORE_DIR}/container_node_pool &&
-    echo "Current directory: $(pwd)" &&
-    terraform init &&
-    terraform plan -input=false -out=tfplan &&
-    terraform apply -input=false tfplan || exit 1
-rm tfplan
-
-cd ${ACP_PLATFORM_CORE_DIR}/gke_enterprise/fleet_membership &&
-    echo "Current directory: $(pwd)" &&
-    terraform init &&
-    terraform plan -input=false -out=tfplan &&
-    terraform apply -input=false tfplan || exit 1
-rm tfplan
-
-# b/376312292
-# cd ${ACP_PLATFORM_CORE_DIR}/gke_enterprise/servicemesh &&
-#     echo "Current directory: $(pwd)" &&
-#     terraform init &&
-#     terraform plan -input=false -out=tfplan &&
-#     terraform apply -input=false tfplan || exit 1
-# rm tfplan
-
-cd ${ACP_PLATFORM_CORE_DIR}/workloads/kueue &&
-    echo "Current directory: $(pwd)" &&
-    terraform init &&
-    terraform plan -input=false -out=tfplan &&
-    terraform apply -input=false tfplan || exit 1 &&
+for terraservice in "${terraservices[@]}"; do
+    cd "${ACP_PLATFORM_CORE_DIR}/${terraservice}" &&
```

Comment:
Nit: this will change the working directory in future commands.

Worth using the `-chdir` Terraform option instead?

Reply:
We try to always use absolute paths and every set of commands should always set their expected directory before running any commands. Using `-chdir` is required for each subcommand.